velox/common/memory/tests/MemoryAllocatorTest.cpp

```diff
@@ -123,25 +123,7 @@ class MemoryAllocatorTest : public testing::TestWithParam<int> {
   /// Returns the virtual and resident sizes of the process in 4K pages. Only
   /// defined for Linux.
   std::optional<ProcessSize> processSize() {
```

Comment:
Can we revert these changes?

Reply:
oh god! @majetideepak It for unblock local build but I shouldn't be committed. Sorry for this mistake. Looks I wasn't focused enough on coding at home during the Spring Festival holiday.

Reverted